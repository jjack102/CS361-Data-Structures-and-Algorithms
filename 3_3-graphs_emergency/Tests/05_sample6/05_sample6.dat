0 5 1
0 3 7 0
3 4 6 1
4 5 17 0
2 3 18 0
2 4 11 0
0 4 14 0
