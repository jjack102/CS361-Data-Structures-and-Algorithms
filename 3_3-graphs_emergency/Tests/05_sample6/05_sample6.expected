Fastest time for car at 0 to the emergency at 5 is 30 minutes.
